shell/browser/browser_win.cc

```diff
@@ -184,18 +185,20 @@ bool FormatCommandLineString(std::u16string* exe,
 std::vector<Browser::LaunchItem> GetLoginItemSettingsHelper(
     base::win::RegistryValueIterator* it,
     boolean* executable_will_launch_at_login,
-    std::u16string scope,
+    std::wstring scope,
     const Browser::LoginItemSettings& options) {
   std::vector<Browser::LaunchItem> launch_items;
 
   base::FilePath lookup_exe_path;
   if (options.path.empty()) {
-    std::u16string process_exe_path;
+    std::wstring process_exe_path;
     GetProcessExecPath(&process_exe_path);
     lookup_exe_path =
         base::CommandLine::FromString(process_exe_path).GetProgram();
   } else {
-    lookup_exe_path = base::CommandLine::FromString(options.path).GetProgram();
+    lookup_exe_path =
+        base::CommandLine::FromString(base::UTF16ToWide(options.path))
```

Comment:
should `LoginItemSettings::path` be `std::wstring`?

Reply:
9127cff58b6740bcf5dffff420ffeb14b24a245f